### PR TITLE
A layout post

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description
Uncomment if you want to provide a custom description.

This needs an image, and some workshop marketing.

## Related Issue(s)
_Reminder to add related issue(s), if available._

- #599

### CODE REVIEW

File: content/blog/2024/css-layout.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+## Large sites (and core features) are slow to change
+
+Many developers are working in a context
+with much more conservative legacy support policies --
+often waiting for 'full' support of a new feature
+before they even consider using it in production.
+
+In conversations,
+I'll especially hear concerns
```
Comment (jgerigmeyer) [inline content/blog/2024/css-layout.md:127]:
```suggestion
I especially hear concerns
```

File: content/blog/2024/css-layout.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+about browsers like iOS Safari
+that update more slowly,
+part of a much heavier OS update.
+Safari on mac may update in monthly-ish cycles,
```
Comment (jgerigmeyer) [inline content/blog/2024/css-layout.md:131]:
```suggestion
Safari on Mac may update in monthly-ish cycles,
```

File: content/blog/2024/css-layout.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+it's another thing to replace
+the entire hull of a ship.
+Even with a smaller team,
+a smaller project,
+and a lighter browser-support matrix --
+it doesn't make sense to re-write your code base
+or design system
+every time a new feature comes out.
+
+We started phasing-out Susy
```
Comment (jgerigmeyer) [inline content/blog/2024/css-layout.md:166]:
```suggestion
We started phasing out Susy
```

File: content/blog/2024/css-layout.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+we were
+[recommending others do the same](http://localhost:8080/talks/no-grid-system/),
+and we
+[officially deprecated](https://github.com/oddbird/susy/commit/018b5c638ec0726d172af96828e6f1712682283b)
+the project
+on Jul 14, 2020.
+Four years later,
+[OddBird.net/susy](/susy/)
+still has the highest traffic of anything on our site,
+the `susy` package gets
```
Comment (jgerigmeyer) [inline content/blog/2024/css-layout.md:179]:
```suggestion
and the `susy` package gets
```

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
The [Learn CSS Grid, Seriously](https://deploy-preview-621--oddleventy.netlify.app/2024/06/13/css-layout/#learn-css-grid%2C-seriously) subtitle should probably be Learn CSS grid, seriously to match the other subtitles.

Can you add a blurb about the CSS Workshop just like you did in the last article you release?

Otherwise, LGTM

File: content/blog/2024/css-layout.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+You'll be fine waiting for
+_widely available_ support
+and your next schedule re-factor.
+
+But if you're still avoiding grid --
+whatever your reasons --
+you are, in fact, _missing out_.
+CSS grid is one of the best features in CSS,
+and one of the biggest time-savers
+on every site we build.
```
Comment (stacyk) [inline content/blog/2024/css-layout.md:196]: 👏🏻

**Review by stacyk — APPROVED** [review]
Love it, really puts things into a priority order for those who have "things to learn" list

File: content/blog/2024/css-layout.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+[officially deprecated](https://github.com/oddbird/susy/commit/018b5c638ec0726d172af96828e6f1712682283b)
+the project
+on Jul 14, 2020.
+Four years later,
+[OddBird.net/susy](/susy/)
+still has the highest traffic of anything on our site,
+the `susy` package gets
+[nearly 19k weekly downloads on npm](https://www.npmjs.com/package/susy).
+
+## Learn CSS Grid, Seriously
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:182]:
```suggestion
## Learn CSS grid (yes, you need it)
```

File: content/blog/2024/css-layout.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+
+There's no rush to rip out
+all your media queries,
+and replace them with containers.
+You'll be fine waiting for
+_widely available_ support
+and your next schedule re-factor.
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:189]:
```suggestion
and your next scheduled re-factor.
```

File: content/blog/2024/css-layout.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+whatever your reasons --
+you are, in fact, _missing out_.
+CSS grid is one of the best features in CSS,
+and one of the biggest time-savers
+on every site we build.
+
+Yes, Flexbox is also pretty cool.
+But flexbox is designed for
+content-out distribution,
+not design-system page layout.
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:201]:
```suggestion
not system-wide page layouts.
```

**Comment by mirisuzanne** [comment]
@SondraE @stacyk @jgerigmeyer - I added the workshop mentions, and a couple other bits of info. Let me know if you see issues, or feel free to merge.

File: content/blog/2024/css-layout.md [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+content-out distribution,
+not system-wide page layouts.
+Using flexbox alone
+is like only using `inline` text,
+without paragraphs, divs, and other `block` elements.
+You're using half the system,
+and _it's the more complicated half to manage_.
+
+At some point
+people looked at Grid and thought
```
Comment (dvdherron) [inline content/blog/2024/css-layout.md:268]:
```suggestion
people looked at grid and thought
```
I think grid is lowercase in most instances in the post. Maybe we capitalize on in section headings or when it's CSS Grid?

**Review by dvdherron — APPROVED** [review]
LGTM: Great way to put things into perspective. Small nit on the consistency of the capitalization of grid.

**Comment by mirisuzanne** [comment]
> Small nit on the consistency of the capitalization of grid.

There's probably no good reason to capitalize outside the names of specification modules?

**Review by stacyk — APPROVED** [review]
In our styleguide, we do say to write headings in Title Case, but I'm not sure we do that consistently

File: content/blog/2024/css-layout.md [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+  those support numbers will never go over 97% for new features.
+  I recommend opening the
+  [CanIUse default display settings](https://caniuse.com/ciu/settings#display),
+  and selecting _All tracked users_.
+  You might also want to apply the 'filtered' view,
+  to see only browsers with more than
+  0.5% usage by default.
+{% endcallout %}
+
+## We can't measure time in features released
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:128]:
```suggestion
## We Can't Measure Time in Features Released
```

File: content/blog/2024/css-layout.md [hunk 11]
```diff
@@ -0,0 +1,286 @@
...
+[officially deprecated](https://github.com/oddbird/susy/commit/018b5c638ec0726d172af96828e6f1712682283b)
+the project
+on Jul 14, 2020.
+Four years later,
+[OddBird.net/susy](/susy/)
+still has the highest traffic of anything on our site,
+and the `susy` package gets
+[nearly 19k weekly downloads on npm](https://www.npmjs.com/package/susy).
+
+## Learn CSS grid (yes, you need it)
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:235]:
```suggestion
## Learn CSS Grid (Yes, You Need It)
```

File: content/blog/2024/css-layout.md [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+No!
+Stop.
+_It's time to learn grid. Right now._
+You might not think you need it,
+but you do.
+Set down container queries,
+and go learn grid.
+
+(If you want to learn more,
+check out our workshop below)
```
Comment (mirisuzanne) [inline content/blog/2024/css-layout.md:286]:
```suggestion
If you want to learn more,
check out our workshop below.
```

**Comment by mirisuzanne** [comment]
@stacyk shoot, I'm really bad at title case. Do my proposed changes look right?

Why is the mixin article showing up as new in this PR?

**Comment by stacyk** [comment]
> @stacyk shoot, I'm really bad at title case. Do my proposed changes look right?
> 
> Why is the mixin article showing up as new in this PR?

we were just talking about the sace during standup now. We decided post titles should be titlecase, and as long as the casing is consistent within the post for headings, then that's fine. 

I also thought it was strange to see the mixin article here. I didn't look into it though.

**Comment by mirisuzanne** [comment]
@stacyk ok, I think it's all fixed!

**Pull request merged**